### PR TITLE
SCSS Re-write, Component extraction and some minor changes

### PR DESCRIPTION
**Pull request by snacksncode**
This is a follow up for an old PR, everything should be fixed if not comment on the issue

### CODE REVIEW

File: components/BlogEntry/BlogEntry.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+//external
```
Comment (Darex1991) [inline components/BlogEntry/BlogEntry.js:1]: ✂️

File: components/BlogEntry/blogEntryStyles.scss [hunk 2]
```diff
@@ -0,0 +1,42 @@
+//variables import
```
Comment (Darex1991) [inline components/BlogEntry/blogEntryStyles.scss:1]: ✂️

File: components/BlogEntry/blogEntryStyles.scss [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+@import '../../styles/variables';
+//main
```
Comment (Darex1991) [inline components/BlogEntry/blogEntryStyles.scss:3]: ✂️

File: components/BlogEntry/blogEntryStyles.scss [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+.blog_entry__wrapper {
```
Comment (Darex1991) [inline components/BlogEntry/blogEntryStyles.scss:4]: `blog-entry`

File: components/BlogEntry/blogEntryStyles.scss [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    width: 100%;
+    margin: 32px 0;
+    padding: 0 16px;
+    text-align: center;
+}
+.blog_entry__image {
```
Comment (Darex1991) [inline components/BlogEntry/blogEntryStyles.scss:10]:
```
.blog_entry {
  &__wrapper {
    ...
  }
  &__image {
    ...
  }
}
```
etc

File: components/BlogEntry/blogEntryStyles.scss [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+    width: 100%;
+    height: auto;
+    margin-bottom: 10px;
+}
+.blog_entry__date {
+    font-size: 14px;
```
Comment (Darex1991) [inline components/BlogEntry/blogEntryStyles.scss:16]: please use variables

File: components/BlogEntry/blogEntryStyles.scss [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+//main
+.blog_entry__wrapper {
+    width: 100%;
+    margin: 32px 0;
+    padding: 0 16px;
+    text-align: center;
+}
+.blog_entry__image {
+    width: 100%;
+    height: auto;
```
Comment (Darex1991) [inline components/BlogEntry/blogEntryStyles.scss:12]: please reorder properties alphabetically

File: components/BlogEntry/blogEntryStyles.scss [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+}
+.blog_entry__button {
+    border: none;
+    border-radius: 4px;
+    cursor: pointer;
+    background-color: $accent;
+    color: $txt_color_light;
+    padding: 8px 16px;
+    font-weight: 700;
+    &:hover,
```
Comment (Darex1991) [inline components/BlogEntry/blogEntryStyles.scss:38]: please add here some space

File: components/Footer/Footer.js [hunk 9]
```diff
@@ -1,29 +1,29 @@
 //components imports
```
Comment (Darex1991) [inline components/Footer/Footer.js:1]: ✂️

File: components/Footer/footerStyles.scss [hunk 10]
```diff
@@ -1,14 +1,17 @@
 //scss variables import
```
Comment (Darex1991) [inline components/Footer/footerStyles.scss:1]: ✂️

File: components/Footer/footerStyles.scss [hunk 11]
```diff
@@ -1,14 +1,17 @@
...
 @import '../.././styles/variables';
 //main element
```
Comment (Darex1991) [inline components/Footer/footerStyles.scss:3]: ✂️

File: components/Footer/footerStyles.scss [hunk 12]
```diff
@@ -1,14 +1,17 @@
...
     display: flex;
     flex-direction: column;
     background-color: $bgc_dark;
 }
+.footer__wrapper .social_icons__wrapper { //some small whitespace fixes
+    margin-top: 16px;
+}
 //width capping
-.container {
+.footer__container {
```
Comment (Darex1991) [inline components/Footer/footerStyles.scss:14]: please nest all

File: components/FooterBlogEntry/footerBlogEntryStyles.scss [hunk 13]
```diff
@@ -1,39 +1,39 @@
 //variables import
```
Comment (Darex1991) [inline components/FooterBlogEntry/footerBlogEntryStyles.scss:1]: ✂️

File: components/FooterBlogEntry/footerBlogEntryStyles.scss [hunk 14]
```diff
@@ -1,39 +1,39 @@
...
 @import '../../styles/variables';
 //main component
```
Comment (Darex1991) [inline components/FooterBlogEntry/footerBlogEntryStyles.scss:3]: ✂️

File: components/FooterBlogEntry/footerBlogEntryStyles.scss [hunk 15]
```diff
@@ -1,39 +1,39 @@
...
-        &__img {
-            width: 100%;
-            vertical-align: middle;
-            display: inline-block;
-        }
-    }
-    .blog__info {
-        width: 60%;
+}
+.footer_blog_entry__image_container {
```
Comment (Darex1991) [inline components/FooterBlogEntry/footerBlogEntryStyles.scss:12]: `footer-blog-entry`

Comment (Darex1991) [reply components/FooterBlogEntry/footerBlogEntryStyles.scss:12]:
```
.footer_blog_entry {
  &__image {
  ...
  
  &-container {
    ..
  }
}

File: components/HamburgerMenu/HamburgerMenu.js [hunk 16]
```diff
@@ -1,14 +1,14 @@
 //moduled stylesheet
-import hamburgerMenuStyles from './HamburgerMenu.module.scss'
+const CLASS_NAME = 'hamburger_menu';
```
Comment (Darex1991) [inline components/HamburgerMenu/HamburgerMenu.js:2]: hamburger-menu

File: components/BlogEntry/BlogEntry.js [hunk 17]
```diff
@@ -0,0 +1,27 @@
...
+import Link from 'next/link'
+
+const CLASS_NAME = 'blog_entry';
```
Comment (Darex1991) [inline components/BlogEntry/BlogEntry.js:4]: `blog-entry`

File: components/Footer/Footer.js [hunk 18]
```diff
@@ -43,9 +43,9 @@ const Footer = () => (
           blogDate="30 december 2006"/>
       </div>
     </div>
-    <div className={footerStyles.copyright}>
-      <div className={footerStyles.copyright__content}>
-        <div className={footerStyles.copyright__text}>Copyright © Selleo 2020. Wszelkie prawa zastrzeżone.</div>
+    <div className={`${CLASS_NAME}__copyright`}>
+      <div className={`${CLASS_NAME}__copyright__content`}>
```
Comment (Darex1991) [inline components/Footer/Footer.js:47]: `copyright-content`

Comment (Darex1991) [reply components/Footer/Footer.js:47]: please try to avoid using double nestings

File: components/Footer/footerStyles.scss [hunk 19]
```diff
@@ -1,14 +1,17 @@
...
 @import '../.././styles/variables';
 //main element
-.wrapper {
+.footer__wrapper {
     color: $txt_color_light;
     display: flex;
     flex-direction: column;
     background-color: $bgc_dark;
 }
+.footer__wrapper .social_icons__wrapper { //some small whitespace fixes
```
Comment (Darex1991) [inline components/Footer/footerStyles.scss:10]: please remove comment and nest this inside parent

File: components/FooterBlogEntry/footerBlogEntryStyles.scss [hunk 20]
```diff
@@ -1,39 +1,39 @@
...
         margin-bottom: 2px;
     }
-    .blog__date {
+    &__date {
         font-size: 12px;
     }
 }
 @media ( min-width: $max_width_cap ) {
-    .wrapper__image_container {
+    .footer_blog_entry__image_container {
```
Comment (Darex1991) [inline components/FooterBlogEntry/footerBlogEntryStyles.scss:36]: `footer-blog-entry__image-container`

File: components/HamburgerMenu/HamburgerMenu.js [hunk 21]
```diff
@@ -1,14 +1,14 @@
...
 
 const HamburgerMenu = ({ active, handleClick }) => (
     <div 
         onClick={handleClick}
-        className={`${hamburgerMenuStyles.wrapper} ${ active === true ? hamburgerMenuStyles['wrapper--active'] : '' }`}
+        className={`${CLASS_NAME}__wrapper ${ active === true ? `${CLASS_NAME}__wrapper--active` : '' }`}
```
Comment (Darex1991) [inline components/HamburgerMenu/HamburgerMenu.js:7]: please use classnames https://www.npmjs.com/package/classnames

File: components/HamburgerMenu/hamburgerMenuStyles.scss [hunk 22]
```diff
@@ -1,7 +1,7 @@
 //variables import
 @import '../../styles/variables';
 //main
-.wrapper {
+.hamburger_menu__wrapper {
```
Comment (Darex1991) [inline components/HamburgerMenu/hamburgerMenuStyles.scss:4]: `hamburger-menu`

File: components/HamburgerMenu/hamburgerMenuStyles.scss [hunk 23]
```diff
@@ -18,7 +18,7 @@
     z-index: 100;
     &--active {
         background: $red;
-        .bar {
+        .hamburger_menu__bar {
```
Comment (Darex1991) [inline components/HamburgerMenu/hamburgerMenuStyles.scss:21]:
```
.hamburger-menu {
  $this: &;

  &--active {
    background: $red;

    #{$this}__bar {}
  }
}

File: components/HamburgerMenu/hamburgerMenuStyles.scss [hunk 24]
```diff
@@ -32,7 +32,7 @@
         }
     }
 }
-.bar { //will result in HamburgerMenu__bar
+.hamburger_menu__bar { //will result in HamburgerMenu__bar
```
Comment (Darex1991) [inline components/HamburgerMenu/hamburgerMenuStyles.scss:35]: please remove comment and nest it

File: components/Header/Header.js [hunk 25]
```diff
@@ -21,7 +20,7 @@ const Header = () => {
   
   return (
     <>
-      <div className={`${headerStyles.wrapper} ${menuOpened ? headerStyles['wrapper--opened'] : ''}`}>
+      <div className={`${CLASS_NAME}__wrapper ${menuOpened ? `${CLASS_NAME}__wrapper--opened` : ''}`}>
```
Comment (Darex1991) [inline components/Header/Header.js:23]: please use classnames

File: components/Hero/Hero.js [hunk 26]
```diff
@@ -10,10 +9,10 @@ const Hero = ({
   titleText
 }) => {
   return (
-      <div className={heroStyles.wrapper}>
-        { showAccent && <div className={heroStyles.accent__text}>{accentText}</div> }
-        { showTitle && <div className={heroStyles.main__text}>{titleText}</div> }
-        { showSubtitle && <div className={heroStyles.secondary__text}>{subtitleText}</div> }
+      <div className={`${CLASS_NAME}__wrapper`}>
+        { showAccent && <div className={`${CLASS_NAME}__accent_text`}>{accentText}</div> }
```
Comment (Darex1991) [inline components/Hero/Hero.js:13]: `__accent-text`

File: components/Hero/Hero.js [hunk 27]
```diff
@@ -10,10 +9,10 @@ const Hero = ({
...
+        { showTitle && <div className={`${CLASS_NAME}__main_text`}>{titleText}</div> }
```
Comment (Darex1991) [inline components/Hero/Hero.js:14]: `__main-text`

File: components/Hero/Hero.js [hunk 28]
```diff
@@ -10,10 +9,10 @@ const Hero = ({
...
+        { showSubtitle && <div className={`${CLASS_NAME}__secondary_text`}>{subtitleText}</div> }
```
Comment (Darex1991) [inline components/Hero/Hero.js:15]: `__secondary-text`

File: components/Hero/heroStyles.scss [hunk 29]
```diff
@@ -13,26 +13,26 @@
     justify-content: center;
     flex-direction: column;
 }
-.accent__text {
+.hero__accent_text {
```
Comment (Darex1991) [inline components/Hero/heroStyles.scss:16]: please nest everything in .hero {}

File: components/Layout/Layout.js [hunk 30]
```diff
@@ -2,21 +2,22 @@
...
+    showAccent,
+    showHero, 
+    showSubtitle,
+    showTitle,
+    subtitleText,
+    titleText
 }) => {
     return (
-        <div className={`${layoutStyles.container} ${!showHero ? layoutStyles['container--heroHidden'] : ''}`}>
+        <div className={`${CLASS_NAME}__container ${!showHero ? `${CLASS_NAME}__container--hero_hidden` : ''}`}>
```
Comment (Darex1991) [inline components/Layout/Layout.js:20]: please use classnames

File: components/Layout/layoutStyles.scss [hunk 31]
```diff
@@ -1,12 +1,12 @@
 @import '../../styles/variables';
-.container {
+.layout__container {
```
Comment (Darex1991) [inline components/Layout/layoutStyles.scss:2]: please add empty line before

File: components/Layout/layoutStyles.scss [hunk 32]
```diff
@@ -1,12 +1,12 @@
...
     min-height: 100vh;
     height: auto;
-    &--heroHidden {
+    &--hero_hidden {
```
Comment (Darex1991) [inline components/Layout/layoutStyles.scss:5]: `hero-hidden`

File: components/Layout/layoutStyles.scss [hunk 33]
```diff
@@ -1,12 +1,12 @@
...
         margin-top: 50px;
     }
 }
-.page__content {
+.layout__page_content {
```
Comment (Darex1991) [inline components/Layout/layoutStyles.scss:9]: please nest it in `layout` and `page-content`

File: components/Navigation/Navigation.js [hunk 34]
```diff
@@ -1,10 +1,9 @@
 //external
```
Comment (Darex1991) [inline components/Navigation/Navigation.js:1]: please remove comments

File: components/Navigation/navigationStyles.scss [hunk 35]
```diff
@@ -1,7 +1,6 @@
-//variables import
 @import '../../styles/variables';
 //main
-.wrapper {
+.navigation__wrapper {
```
Comment (Darex1991) [inline components/Navigation/navigationStyles.scss:3]: please nest it in `navigation`

File: components/SocialIcons/socialIconsStyles.scss [hunk 36]
```diff
@@ -1,11 +1,11 @@
-.wrapper {
+.social_icons__wrapper {
```
Comment (Darex1991) [inline components/SocialIcons/socialIconsStyles.scss:1]: same here

File: data/blogData.js [hunk 37]
```diff
@@ -0,0 +1,26 @@
...
+    },
+    {
+        "image": "https://source.unsplash.com/1600x900/?cat",
+        "imageAlt": "Obrazek się nie załadował",
+        "date": "4 marca 2019",
+        "title": "Cats, cats, cats",
+        "subtitle": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum",
+        "link": "/"
+    }
+]
```
Comment (Darex1991) [inline data/blogData.js:26]: please set your IDE with empty line on the bottom of file

File: pages/index.js [hunk 38]
```diff
@@ -9,14 +13,20 @@ const IndexPage = () => {
...
-        <hr/>
-        <Link href="/about">
-            <a>Link to about page</a>
-        </Link>
+        <h1 className={`${CLASS_NAME}__title`}>Recent Posts</h1>
+        {
+          Object.values(blogData).map((entry, index) => (
+            <BlogEntry
+              key={`${entry.title}_${index}`}
+              blogImage={entry.image}
```
Comment (Darex1991) [inline pages/index.js:21]: please reorder these alphabetically

File: styles/contactPageStyles.scss [hunk 39]
```diff
@@ -1,5 +1,5 @@
 @import './variables.scss';
-.title {
+.contact__title {
```
Comment (Darex1991) [inline styles/contactPageStyles.scss:2]: please add empty line before

File: styles/homePageStyles.scss [hunk 40]
```diff
@@ -0,0 +1,14 @@
+@import './variables.scss';
+.home_page__title {
+    font-size: 32px;
```
Comment (Darex1991) [inline styles/homePageStyles.scss:3]: please use variables

File: styles/homePageStyles.scss [hunk 41]
```diff
@@ -0,0 +1,14 @@
+@import './variables.scss';
+.home_page__title {
```
Comment (Darex1991) [inline styles/homePageStyles.scss:2]: please add empty line before

File: styles/homePageStyles.scss [hunk 42]
```diff
@@ -0,0 +1,14 @@
...
+    font-size: 32px;
+    margin-bottom: 16px;
+    text-align: center;
+    &::after {
```
Comment (Darex1991) [inline styles/homePageStyles.scss:6]: `&:after`

Comment (Darex1991) [reply styles/homePageStyles.scss:6]: and please add empty line before

File: styles/homePageStyles.scss [hunk 43]
```diff
@@ -0,0 +1,14 @@
...
+        content: "";
+        display: block;
+        width: 64px;
+        height: 4px;
```
Comment (Darex1991) [inline styles/homePageStyles.scss:10]: please reorder these alphabetically

File: components/HamburgerMenu/hamburgerMenuStyles.scss (hunk 23 above)
Comment (snacksncode) [reply components/HamburgerMenu/hamburgerMenuStyles.scss:21]: :0 thats some advanced ampersand usage. Never would've thought of that

File: styles/homePageStyles.scss (hunk 42 above)
Comment (snacksncode) [reply styles/homePageStyles.scss:6]: :after is a very old syntaxis. Are you sure?

Comment (Darex1991) [reply styles/homePageStyles.scss:7]: yeah

File: components/BlogEntry/blogEntryStyles.scss [hunk 44]
```diff
@@ -0,0 +1,50 @@
+@import '../../styles/variables';
+
+.blog-entry {
+
```
Comment (Darex1991) [inline components/BlogEntry/blogEntryStyles.scss:4]: ✂️

File: components/BlogEntry/blogEntryStyles.scss [hunk 45]
```diff
@@ -0,0 +1,50 @@
...
+    }
+
+    &__image {
+        margin-bottom: 10px;
+        width: 100%;
+        height: auto;
+    }
+
+    &__date {
+        font-size: $fs_small;
```
Comment (Darex1991) [inline components/BlogEntry/blogEntryStyles.scss:19]: once again. Please check boilerplate and check how name variables https://selleo.slack.com/files/U6UM94QEB/F014U35UXL2/sass-boilerplate-master.zip

File: components/BlogEntry/blogEntryStyles.scss [hunk 46]
```diff
@@ -0,0 +1,50 @@
...
+
+    &__button {
+        border: none;
+        border-radius: 4px;
+        cursor: pointer;
+        background-color: $accent;
+        color: $txt_color_light;
+        padding: 8px 16px;
+        font-weight: 700;
+        &:hover,
```
Comment (Darex1991) [inline components/BlogEntry/blogEntryStyles.scss:44]: please add space before

File: components/Footer/Footer.js [hunk 47]
```diff
@@ -1,29 +1,28 @@
...
-  <div className={footerStyles.wrapper}>
-    <div className={footerStyles.container}>
-      <div className={footerStyles.column}>
-        <div className={footerStyles.logo__wrapper}>
-          <img className={footerStyles.logo__wrapper__image} src="/logo.svg" />
+  <div className={`${CLASS_NAME}__wrapper`}>
+    <div className={`${CLASS_NAME}__container`}>
+      <div className={`${CLASS_NAME}__column`}>
+        <div className={`${CLASS_NAME}__logo-wrapper`}>
+          <img className={`${CLASS_NAME}__logo-wrapper__image`} src="/logo.svg" />
```
Comment (Darex1991) [inline components/Footer/Footer.js:11]: `${CLASS_NAME}__logo` please avoid double nestings in bem

File: components/Footer/footerStyles.scss [hunk 48]
```diff
@@ -0,0 +1,97 @@
+@import '../.././styles/variables';
+
+.footer {
+
```
Comment (Darex1991) [inline components/Footer/footerStyles.scss:4]: ✂️

File: components/Footer/footerStyles.scss [hunk 49]
```diff
@@ -0,0 +1,97 @@
...
+    &__wrapper {
+        color: $txt_color_light;
+        display: flex;
+        flex-direction: column;
+        background-color: $bgc_dark;
+        .social_icons__wrapper {
+            margin-top: 16px;
+        }
+    }
+    //width capping
```
Comment (Darex1991) [inline components/Footer/footerStyles.scss:14]: please remove all comments

File: components/Footer/footerStyles.scss [hunk 50]
```diff
@@ -0,0 +1,97 @@
+@import '../.././styles/variables';
+
+.footer {
+
+    &__wrapper {
+        color: $txt_color_light;
+        display: flex;
+        flex-direction: column;
+        background-color: $bgc_dark;
+        .social_icons__wrapper {
```
Comment (Darex1991) [inline components/Footer/footerStyles.scss:10]: please add space before and reorder properties - on all scss files

File: components/Footer/footerStyles.scss [hunk 51]
```diff
@@ -0,0 +1,97 @@
...
+            margin-top: 16px;
+        }
+    }
+    //width capping
+    &__container {
+        max-width: $max_width_cap;
```
Comment (Darex1991) [inline components/Footer/footerStyles.scss:16]: `$max-width-cap`

File: components/Footer/footerStyles.scss [hunk 52]
```diff
@@ -0,0 +1,97 @@
...
+        flex-direction: column;
+    }
+    
+    &__copyright {
+        width: 100%;
+        display: flex;
+        background-color: $bgc_darker;
+    }
+    
+    &__copyright-content {
```
Comment (Darex1991) [inline components/Footer/footerStyles.scss:36]: please nest it inside `__copyright`

File: components/Footer/footerStyles.scss [hunk 53]
```diff
@@ -0,0 +1,97 @@
...
+        max-width: $max_width_cap;
+        margin: 0 auto;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        width: 100%;
+        align-items: center;
+        padding: 16px;
+        text-align: center;
+        .copyright__text {
```
Comment (Darex1991) [inline components/Footer/footerStyles.scss:46]: why you nested it here? this is not modifier

File: components/Footer/footerStyles.scss [hunk 54]
```diff
@@ -0,0 +1,97 @@
...
+        margin-bottom: 16px;
+    }
+
+    &__title {
+        font-size: 24px;
+        text-align: center;
+        font-weight: 700;
+        margin: 16px 0;
+    }
+
```
Comment (Darex1991) [inline components/Footer/footerStyles.scss:61]: ✂️

File: components/Footer/footerStyles.scss [hunk 55]
```diff
@@ -0,0 +1,97 @@
...
+}
```
Comment (Darex1991) [inline components/Footer/footerStyles.scss:62]: please add empty line here and remove one between media query and .footer

File: components/Footer/footerStyles.scss [hunk 56]
```diff
@@ -0,0 +1,97 @@
...
+            margin: 0;
+            display: flex;
+            text-align: left;
+            align-items: center;
+        }
+       
+        &__logo-wrapper {
+            height: 100px;
+            width: auto;
+            &__image {
```
Comment (Darex1991) [inline components/Footer/footerStyles.scss:86]: it shouldn't be nested

File: components/FooterBlogEntry/FooterBlogEntry.js [hunk 57]
```diff
@@ -1,14 +1,13 @@
...
-            <img className={footerBlogStyles.wrapper__image_container__img} src={imageSrc} alt={imageAlt}/>
+    <div className={`${CLASS_NAME}__wrapper`}>
+        <div className={`${CLASS_NAME}__image-container`}>
+            <img className={`${CLASS_NAME}__image-container__img`} src={imageSrc} alt={imageAlt}/>
         </div>
-        <div className={footerBlogStyles.blog__info}>
-            <div className={footerBlogStyles.blog__title}>{blogTitle}</div>
-            <div className={footerBlogStyles.blog__date}>{blogDate}</div>
+        <div className={`${CLASS_NAME}__info`}>
+            <div className={`${CLASS_NAME}__info__title`}>{blogTitle}</div>
```
Comment (Darex1991) [inline components/FooterBlogEntry/FooterBlogEntry.js:9]: without double nesting

File: components/FooterBlogEntry/footerBlogEntryStyles.scss [hunk 58]
```diff
@@ -0,0 +1,40 @@
+@import '../../styles/variables';
+
+.footer-blog-entry {
+    &__wrapper {
+        width: 100%;
+        display: flex;
+        min-height: 1px; //IE 11 quickfix (img aspect ratio)
```
Comment (Darex1991) [inline components/FooterBlogEntry/footerBlogEntryStyles.scss:7]: please remove comments

File: components/FooterBlogEntry/footerBlogEntryStyles.scss [hunk 59]
```diff
@@ -0,0 +1,40 @@
...
+            margin-bottom: 2px;
+        }
+        &__date {
+            font-size: 12px;
+        }
+    }
+}
+@media ( min-width: $max_width_cap ) {
+    .footer-blog-entry__image-container {
+            width: 20% !important;
```
Comment (Darex1991) [inline components/FooterBlogEntry/footerBlogEntryStyles.scss:38]: you have something wrong with you IDE. tab or double spaces as nesting

File: components/HamburgerMenu/hamburgerMenuStyles.scss [hunk 60]
```diff
@@ -0,0 +1,56 @@
...
+        transition: transform 450ms, background 450ms;
+        transition-timing-function: cubic-bezier(0.83, 0, 0.17, 1);
+        &:nth-child(2) {
+            width: 50%;
+        }
+    }
+
+}
+
+@media ( min-width: $max_width_cap ) {
```
Comment (Darex1991) [inline components/HamburgerMenu/hamburgerMenuStyles.scss:54]: $max-width-cap

Comment (Darex1991) [reply components/HamburgerMenu/hamburgerMenuStyles.scss:54]: in boilerplate you will find media query mixins

File: components/Hero/heroStyles.scss [hunk 61]
```diff
@@ -0,0 +1,40 @@
...
+        margin: 12px 0;
+    }
+    &__secondary-text {
+        font-size: $fs_small;
+    }
+}
+@media ( min-width: 768px ) {
+    .hero__wrapper {
+        min-height: 20vh;
+        height: auto; //IE 11 quickfix
```
Comment (Darex1991) [inline components/Hero/heroStyles.scss:38]: comment

File: components/Navigation/navigationStyles.scss [hunk 62]
```diff
@@ -0,0 +1,145 @@
+// this whole section is pretty "hacky" css
```
Comment (Darex1991) [inline components/Navigation/navigationStyles.scss:1]: ✂️

File: components/NavigationLink/NavigationLink.jsx [hunk 63]
```diff
@@ -0,0 +1,38 @@
...
+
+const CLASS_NAME = "navigation"
+
+const NavigationLink = ({linkObject}) => {
+    const [ dropDownActive, setDropDownActive ] = useState(false);
+
+    const toggleDropDown = () => setDropDownActive(oldState => !oldState);
+    
+    return (
+        linkObject.dropDownMenu === true ? (
```
Comment (Darex1991) [inline components/NavigationLink/NavigationLink.jsx:12]: `linkObject.dropDownMenu ?`
or 
`!!linkObject.dropDownMenu ?`

File: components/NavigationLink/NavigationLink.jsx [hunk 64]
```diff
@@ -0,0 +1,38 @@
...
+            <div className={`${CLASS_NAME}__link`}>
+                <a
+                    href="#"
+                    onClick={toggleDropDown}
+                    className={`${CLASS_NAME}__link-text ${CLASS_NAME}__link-text--has-dropdown`}
+                >{linkObject.linkText}</a>
+                
+                <div className={`${CLASS_NAME}__dropdown ${dropDownActive ? `${CLASS_NAME}__dropdown--active` : ''}`}>
```
Comment (Darex1991) [inline components/NavigationLink/NavigationLink.jsx:20]: please use classnames package

File: components/NavigationLink/NavigationLink.jsx [hunk 65]
```diff
@@ -0,0 +1,38 @@
...
+                    { linkObject.dropDownLinks.map((dropDownLink, index) => (
+                        <Link key={`${dropDownLink.linkText}_${index}`} href={dropDownLink.linkTo}>
+                            <a className={`${CLASS_NAME}__dropdown__link`}>{dropDownLink.linkText}</a>
```
Comment (Darex1991) [inline components/NavigationLink/NavigationLink.jsx:23]: without double nesting

File: components/SocialIcons/socialIconsStyles.scss [hunk 66]
```diff
@@ -0,0 +1,35 @@
...
+        width: 20px;
+        height: 20px;
+        display: flex;
+        margin: 0 8px;
+        align-items: center;
+        justify-content: center;
+        a {
+            width: 100%;
+            height: 100%;
+            #{$this}__youtube_icon {
```
Comment (Darex1991) [inline components/SocialIcons/socialIconsStyles.scss:22]: youtube-icon

File: components/SocialIcons/socialIconsStyles.scss [hunk 67]
```diff
@@ -0,0 +1,35 @@
...
+                color: #f44336;
+            }
+            #{$this}__facebook_icon {
```
Comment (Darex1991) [inline components/SocialIcons/socialIconsStyles.scss:25]: facebook-icon

File: components/SocialIcons/socialIconsStyles.scss [hunk 68]
```diff
@@ -0,0 +1,35 @@
...
+        height: 20px;
+        display: flex;
+        margin: 0 8px;
+        align-items: center;
+        justify-content: center;
+        a {
+            width: 100%;
+            height: 100%;
+            #{$this}__youtube_icon {
+                color: #f44336;
```
Comment (Darex1991) [inline components/SocialIcons/socialIconsStyles.scss:23]: variable

File: components/SocialIcons/socialIconsStyles.scss [hunk 69]
```diff
@@ -0,0 +1,35 @@
...
+            }
+            #{$this}__facebook_icon {
+                color: #3f51b5;
```
Comment (Darex1991) [inline components/SocialIcons/socialIconsStyles.scss:26]: variable

File: styles/_variables.scss [hunk 70]
```diff
@@ -6,6 +6,13 @@ $bgc_white: #ffffff;
 $txt_color_dark: #222222;
 $txt_color_light: #eaeaea;
 
+$fs_bigger: 36px;
+$fs_big: 28px;
```
Comment (Darex1991) [inline styles/_variables.scss:10]: please use boilerplate example for all variables!

**Comment by PTISteezy** [comment]
alphabetical scss,
no comments,
JS -> JSX,
font size vars,
new page /patient/ambulance

File: components/SideBar/sideBarStyles.scss [hunk 71]
```diff
@@ -0,0 +1,20 @@
+@import '../../styles/variables';
+
+.sidebar {
+    &__wrapper {
```
Comment (Darex1991) [inline components/SideBar/sideBarStyles.scss:4]: you can merge it with first line
`sidebar__wrapper`

File: components/SideBar/sideBarStyles.scss [hunk 72]
```diff
@@ -0,0 +1,20 @@
...
+        flex-basis: 100%;
+        margin-top: 16px;
+        padding: 8px;
+        width: 100%;
+    }
+}
+
+@media ( min-width: $max_width_cap ) {
+    .sidebar {
+        &__wrapper {
```
Comment (Darex1991) [inline components/SideBar/sideBarStyles.scss:14]: same here

File: components/SideBar/sideBarStyles.scss [hunk 73]
```diff
@@ -0,0 +1,20 @@
...
+            width: 25%;
```
Comment (Darex1991) [inline components/SideBar/sideBarStyles.scss:15]: alphabetic

File: components/SideBarItem/sideBarItemStyles.scss [hunk 74]
```diff
@@ -0,0 +1,40 @@
+@import '../../styles/variables';
+
+.sidebar-item {
+    background: darken($bgc_light, 7.5);
+    border-radius: 4px;
+    overflow: hidden;
```
Comment (Darex1991) [inline components/SideBarItem/sideBarItemStyles.scss:6]: please check everywhere alphabetic

File: styles/patient/transportStyles.scss [hunk 75]
```diff
@@ -0,0 +1,70 @@
...
+.transport {
+    &__important {
+        color: $red;
+        font-size: $font-size-xl;
+        font-weight: 700;
+        text-align: center;
+    }
+    &__images-container {
+        display: flex;
+        width: 100%;
```
Comment (Darex1991) [inline styles/patient/transportStyles.scss:12]: alphabetic

File: styles/patient/transportStyles.scss [hunk 76]
```diff
@@ -0,0 +1,70 @@
...
+        td {
+            font-size: $font-size-sm;
+        }
+    }
+}
+@media (min-width: $max_width_cap ) {
+    .transport {
+        &__images-container {
+            flex-wrap: nowrap;
+            &__image {
```
Comment (Darex1991) [inline styles/patient/transportStyles.scss:59]: &__image:not(:last-child)

File: styles/patient/partyStyles.scss [hunk 77]
```diff
@@ -0,0 +1,34 @@
+@import '../variables';
+
+.party {
+    &__images-container {
+        display: flex;
+        width: 100%;
```
Comment (Darex1991) [inline styles/patient/partyStyles.scss:6]: alphabetic

File: styles/patient/partyStyles.scss [hunk 78]
```diff
@@ -0,0 +1,34 @@
...
+    &__cennik {
+        font-size: $font-size-base;
+        font-weight: 700;
+    }
+}
+@media (min-width: $max_width_cap ) {
+    .party {
+        &__images-container {
+            flex-wrap: nowrap;
+            &__image {
```
Comment (Darex1991) [inline styles/patient/partyStyles.scss:26]: `__image:not(:last-child)`

**Pull request merged**